README.md

```diff
@@ -13,10 +13,11 @@
 - spreadsheet_id (string, required): your spreadsheet's id
 - worksheet_gid (integer, default: 0): worksheet's gid if you want to specific worksheet
 - mode (string, default: append): writing record method, available mode are `append` and `replace`
-- is_write_header (bool, default: false): if true, write header to first record
+- is_write_header_line (bool, default: false): if true, write header to first record
 - start_cell (string, default: 'A1'): specific the index of first record by the A1
   notation.
-- null_representation (string, default: ''): replace null to `null_representation`
+- null_string (string, default: ''): replace null to `null_string`
+- default_timezone (string, default: 'UTC'): time zone of timestamp columns
```

Comment:
I want `default_timestamp_format` too so that we can specify the format.

Reply:
What we need to support is timezone strings like `+00:00`, `+09:00`.
Supporting `UTC` specially is good idea, but we do not need to support something like `CST` because we can not recognize whether they are China Standard Time or (America) Central Standard Time.

We can use `Time#localtime`.
Note that it destructively changes the timezone of the object (although it does not have !), So we write

```
time.dup.localtime("+09:00").strftime("%Y-%m-%d %H:%M:%S.%6N %:z")
```

in the case of filter plugin, but this is an output plugin (the last plugin), it should be okay to destory the time object, thus,

```
time.localtime("+09:00").strftime("%Y-%m-%d %H:%M:%S.%6N %:z")
```